app/vep/utils/vcf_results.py

```diff
@@ -144,117 +147,131 @@ def _get_alt_allele_details(
     )
 
 
+def _get_vcf_meta(vcf_path: FilePath) -> model.VcfMetadata:
```

Comment:
A lot can go wrong here. It would be good to add some validation or throw some meaningful errors.
- bcftools missing
- bcftools output differs slightly from expected - I see it is locked in the docker so probably fine
- variant_count assumes multiple `:` and that the last element on the split is a number
- grep's output can different on different shells, distros - I think you are safe with what you are doing

Reply:
Most runtime failures in this python file are not recoverable (data issues), which will throw an exception, resulting in a 500 response and a line in error log. I've added some more context to exception messages, hopefully it helps with debugging. For the mentions:
* bcftools is installed in dockerfile (passed image build test). Failures in command-line when running bcftools will throw an exception.
* bcftools output is checked by datamodel, e.g. non-ints in the count variables will throw KeyError
* the full (grepped) line from `bcftools stats` looks like `SN 0 number of records: 12345` (unique keywords + single colon). Anything else should result in an exception.